README.md

```diff
@@ -4,10 +4,9 @@ This repository saves and keeps tracking various documents or etc.
 
 #### Who is the board
 
-- Anna Pawlicka [@annapawlicka](https://github.com/annapawlicka)
 - Chris Koehnen [@crkoehnen](https://github.com/crkoehnen)
 - David Chambers [@davidchambers](https://github.com/davidchambers)
-- Malwine Gier [@Malwine](https://github.com/Malwine)
+- Franka [@vsmart](https://github.com/vsmart)
```

Comment:
I've updated the pull request. Please confirm that your details are correct, and merge the pull request if they are. :)